src/cratedb_fivetran_destination/schema_migration_helper.py

```
@@ -0,0 +1,356 @@
+# ruff: noqa: E501,S608
```

Ruff (0.14.6)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)